system/CodeIgniter.php

```diff
@@ -190,7 +191,7 @@ public function initialize()
         }
 
         // Set default locale on the server
-        locale_set_default($this->config->defaultLocale ?? 'en');
+        Locale::setDefault($this->config->defaultLocale ?? 'en');
```

Comment:
I'm curious why you changed this. Isn't they the same?

Reply:
Now we are using `Locale::getDefault()` and I searched `Locale::setDefault()` but not found.
Using `locale_set_default()` and `Locale::getDefault()` is not consistent.